tanmayk/gizra-starter
# Backend dev

PR for backend task.

Steps to review:
1. Run composer to get OG module & import the configurations.
2. Log in as an administrator & Create Group content.
3. Log in as registered user & go to /groups page. This page shows list of available groups. Refer scr-backend-group.png
4. Click on the title of any group & a greeting message should be visible if user is not a member of group. Refer scr-greeting-display.png.
![scr-backend-groups](https://user-images.githubusercontent.com/1642831/168135859-fcd6b5e6-8cf9-4352-9af9-c95ade173e33.png)
![scr-greeting-display](https://user-images.githubusercontent.com/1642831/168798198-0e383f10-ea25-4e9c-bf7e-d221d89dd2f7.png)
![scr-greeting-formatter](https://user-images.githubusercontent.com/1642831/168798286-7e255c1e-0736-4a1c-a2b8-9f02727d841a.png)
